src/main/webapp/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="hideElements()">
+    <header>
+      <a href="index.html"><h1>Black Student Insight</h1></a>
+      <input type="text" id="school-search" placeholder="Find your institution"/>
+      <search-button onclick="loadSchoolInfo();hideElements();">Search</search-button>
```

Comment:
`<search-button>` is not something official in HTML language, you should use `<button>` tag instead.

Reply:
It's actually okay to create a custom tag like this - you do need to define it in JavaScript, though (window.customElements.define(...); see https://css-tricks.com/styling-a-web-component/). I don't know if the linter will be happy with this or if something else needs to be done.

Reply:
oh yes, I thought it was warned by linter but actually not. Linter reported the css selector instead of the custom tag. So we are good here.